pdf con los datos de usuario

rojas1995 opened the pull request:
Ten en cuenta que actualizo el requeriments con la librería  de pdf

Commit:
[feat] Generación de pdf

He hecho una url para la descarga de un pdf que se genera con una plantilla html

Commit:
[feat] Ahora el pdf muestra todos los datos

Commit:
[feat] Añado la libreria de pdf a los requeriments

Commit:
[fix] Ahora el html tiene title y he cambiado el nombre al pdf

El titlees para quitar un bug de html

Commit:
[fix] Alñado el decorador del login

rojas1995 committed:
Merge branch 'dev' into feat/be/CU-26

angbongon commented:
Resuelve los conflictos, parace que tienes una url mal puesta, seguramente una coma. Al resolver los conflictos debería solucionarse.

rojas1995 committed:
Merge branch 'dev' into feat/be/CU-26

rojas1995 commented:
Ya lo he solucionado, era una coma lo que faltaba.

angbongon reviewed (dismissed):
Se aceptan los cambios y se pasa a QA para ver si es necesario añadir más información

angelcalzadollamas reviewed (changes requested):
Con motivo de tenerlo ya preparado, ¿sería complicado añadir también los recibos del usuario? Si se puede hacer ya hagámoslo cuanto antes.

Me he dado cuenta que el botón "descargar datos" en datos personales ahora mismo no lleva a esta funcionalidad. Háblalo con juanmi a ver si lo puedes enlazar ya correctamente.

Commit:
[fix] Peticiones de cambio hechas

Ahora el botón de descargar datos de usuario funciona correctamente. También he añadido las transacciones al pdf.

rojas1995 committed:
Merge branch 'dev' into feat/be/CU-26

angelcalzadollamas reviewed (dismissed):
Todo correcto.

angbongon committed:
Merge branch 'dev' into feat/be/CU-26
